tools/makezcm.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+
+# Convert source audio file into ZCM digi file.
+# Requires python 'sox' module to be installed, and the sox command line tool.
+
+import os
+import sys
+import struct
+import tempfile
+import sox
+import argparse
+
+
+MAX_SIZE = 2048*1024 - 8192    #  2 mb banked ram minus 1 bank (8kb) for kernal.
+
+def convert_source_to_raw(source: str, outputformat: dict):
+    info = sox.file_info.info(source)
+    print("Source file: ",source)
+    print(f"   {info['channels']} channels, {info['bitdepth']} bits, {int(info['sample_rate'])} hz ({round(info['duration'], 3)} sec.)")
+    tfm = sox.Transformer()
+    tfm.contrast()
+    tfm.set_output_format(file_type="raw", encoding = "signed-integer" if outputformat["bits"]>8 else None)
+    tfm.set_globals(dither=True)
+    tfm.lowpass(frequency = outputformat["rate"]/2)
```

Comment:
I don't know if this is useful or that sox may be doing this internally when downsampling